packages/now-cli/src/util/output/table.js

```diff
@@ -1,8 +1,7 @@
 import chalk from 'chalk';
-import printf from 'printf';
 
 const printLine = (data, sizes) =>
-  data.reduce((line, col, i) => line + printf(`%-${sizes[i]}s`, col), '');
+  data.reduce((line, col, i) => line + col.padEnd(sizes[i]), '');
```

Comment:
Since Node 8, we can use [`padEnd()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padEnd) to right pad a string.